main.c

```diff
@@ -75,16 +118,34 @@ int main(int argc, char *argv[])
 		}
 	}
 
-	if (optind >= argc) {
+	if (is_query_has_udev) {
+		exit(!has_udev);
+	}
+
+	if (!is_udev && argc >= optind) {
```

Comment:
Noticed a bug with this condition:

```suggestion
	if (!is_udev && optind >= argc) {
		usage(argv[0]);
	}

	if (is_udev && optind != argc) {
```